.circleci/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    steps:
```

Comment:
Just a useful tidbit - for circle (or other yaml configs) when you have a chunk of commands that you want to run together, you can do:

```yaml
    steps:
      - checkout
      - run:
            name: Docker Deploy
            command: |
               docker login -u $DOCKER_USER -p $DOCKER_PASS
               docker build -t poldracklab/pydeface:$CIRCLE_BUILD_NUM-${CIRCLE_SHA1:0:6} -f Dockerfile .
               docker run -it poldracklab/pydeface pytest pydeface
               docker tag poldracklab/pydeface:$CIRCLE_BUILD_NUM-${CIRCLE_SHA1:0:6} poldracklab/pydeface:latest
               docker push poldracklab/pydeface
```

What's the difference? When you have a single `run` it invokes one shell, and defaults to use the line as both the command and the name. In short, each run line invokes a new shell. What the above would do is allocate the entire contents of the command to be run in the same shell, and under a common name (as provided above). If you are interested to learn more, [read here](https://circleci.com/docs/2.0/configuration-reference/#run).

Reply:
Thanks I'll update that.